teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, name: 'unknown', parent_permission: true, specialization: 'unknown')
+    super(age, name, parent_permission)
```

Comment:
- [ ] To avoid any argument error please make sure to pass the parameters using the following way:

```ruby 
super(age, name: name, parent_permission: parent_permission)
```